libs/api/domains/cms/src/lib/models/sidebarCard.model.ts

```diff
@@ -0,0 +1,29 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+import { ISidebarCard } from '../generated/contentfulTypes'
+import { Link, mapLink } from './link.model'
+
+@ObjectType()
+export class SidebarCard {
+  @Field(() => ID)
+  id!: string
+
+  @Field()
+  type!: string
+
+  @Field()
+  title!: string
+
+  @Field()
+  content!: string
+
+  @Field(() => Link, { nullable: true })
+  link!: Link | null
```

Comment:
Not sure what the use case is here but if this link field includes internal links we should consider using ReferenceLink here to allow for dynamic link resolution.
ReferenceLink also supports external link resolution

Reply:
Yes that's in the backlog. We ran into some problems with `ReferenceLink` with a slug and subslug. One solution was to use your UI extension but it didn't always seem to update the slug correctly.

Maybe this deserves its own PR? :smile:

Reply:
Yes, also seems not to belong to this PR anymore 😄